lib/registration.pm

```diff
@@ -680,7 +680,7 @@ sub install_docker_when_needed {
     }
     else {
         if (is_sle('<15')) {
-            assert_script_run('zypper se docker || zypper -n ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/SUSE:SLE-12:Update.repo');
+            assert_script_run('zypper se docker || zypper -n ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/SUSE:SLE-12:Update.repo', 120);
```

Comment:
Is this change intentional?

Reply:
Ou, pardon me - it is but it should be obviously somewhere else :)